hydra-node/src/Hydra/Network/Etcd.hs

```diff
@@ -200,11 +205,19 @@ withEtcdNetwork tracer protocolVersion config callback action = do
         , -- XXX: Could use unique initial-cluster-tokens to isolate clusters
           ["--initial-cluster-token", "hydra-network-1"]
         , ["--initial-cluster", clusterPeers]
-        , -- Keep up to 1000 revisions. See also:
-          -- https://etcd.io/docs/v3.5/op-guide/maintenance/#auto-compaction
-          ["--auto-compaction-mode=revision", "--auto-compaction-retention=1000"]
         ]
 
+  onlyEtcdVars = Map.filterWithKey (\k _ -> "ETCD_" `isPrefixOf` k)
```

Comment:
Why are you doing this filtering? The default before was to inherit the whole environment.